### ISSUE TITLE
Remove oca_dependencies.txt and CONTRIBUTING.md from the template?

### ISSUE THREAD

**Issue by sbidoul** [OCA/oca-addons-repo-template#10]
Should we remove `oca_dependencies.txt` from this template? Otherwise `copier update` will overwrite it.

**Comment by sbidoul** [comment]
There is a similar issue with `CONTRIBUTING.md` which has a `Project Specific Guidelines` section for which there is no user question, so we can't update it automatically.

**Comment by Yajo** [comment]
Well, we have several options.

We could add these files to skip in `copier.yml` if we really want Copier to ignore them if present: https://copier.readthedocs.io/en/stable/configuring/#skip_if_exists

However, regarding how these files are right now, Copier will work just fine. When updating, it will smartly detect handmade changes and respect those. There's an update graph in https://copier.readthedocs.io/en/latest/updating/#how-the-update-works  that explains how Copier updates are "smart".

In short, there's nothing to worry about here.
